src/Components/DummyWelcome/DummyWelcome.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import Dummy from "../Dummy/Dummy";
+import Welcome from "../Welcome/Welcome";
```

Comment:
What do you think of naming `index.js` to each file exporting the component itself? So when importing them you don't have to repeat it twice (folder + file)